File: kotlin/raindrops/src/main/kotlin/Raindrops.kt
class Raindrops {
   companion object {
      fun convert(num: Int): String {
         var s = ""
         if(num % 3 == 0) s+= "Pling"
         if(num % 5 == 0) s+= "Plang"
         if(num % 7 == 0) s+= "Plong"
         if(s=="") s = num.toString()
         return s
      }
   }
}

File: python/raindrops/raindrops.py
def convert(number):
    s = ""
    if number % 3 == 0:
        s += "Pling"
    if number % 5 == 0:
        s += "Plang"
    if number % 7 == 0:
        s += "Plong"
    
    return s if s else str(number)